test/common/config/delta_subscription_state_old_test.cc

```diff
@@ -683,18 +683,6 @@ TEST_F(OldVhdsDeltaSubscriptionStateTest, ResourceTTL) {
   EXPECT_CALL(*ttl_timer_, enabled());
   EXPECT_CALL(*ttl_timer_, enableTimer(std::chrono::milliseconds(1000), _));
   deliverDiscoveryResponse(create_resource_with_ttl(true), {}, "debug1", "nonce1", true, 1);
-
-  // Heartbeat update should not be propagated to the subscription callback.
-  EXPECT_CALL(*ttl_timer_, enabled());
-  deliverDiscoveryResponse(create_resource_with_ttl(false), {}, "debug1", "nonce1", true, 0);
```

Comment:
Sorry if this is obvious, but why do you need to remove this call and the one on line 697? Shouldn't you just remove the runtime value merge?

Reply:
Seems it is a mistake, thanks.